src/App.js

```diff
@@ -73,11 +56,8 @@ function App (){
           <div className='header-image' style={{backgroundImage:`url(${iconFile})`
           }}
           />
-
           <div>{htmlHeader}</div>
-        </div>
-        
-            
+        </div>   
           <div className="arrow-up"></div>
         </header>
         <div className='App-main'>
```

Comment:
(1) Eslint issue:
 <div className='App-image' style={{backgroundImage:`url(${ipadMini})`
        }}
        />
        
        It is better to code:
        ` <div
                   className='App-image'
                   style={{backgroundImage:`url(${ipadMini})` }}
          />
        `

Reply:
(2) Eslint issue: <div className="arrow-left"></div>
(3) eslint issue: <div className="arrow-right"></div>
(4) {contentIndex?
            'Back':
            'This is the first page'}
            
            It is better to code like this:
             `{contentIndex
                ? 'Back'
                : 'This is the first page'
               }`